### ISSUE TITLE
cannot unmarhsall json error

### ISSUE THREAD

**Issue by x0rzkov** [shuheiktgw/go-travis#70]
Hi,

Hope you are all well !

I have an error while using this api.

```go
package main

import (
	"context"
	"log"

	"github.com/fatih/structs"
	"github.com/k0kubun/pp"
	"github.com/shuheiktgw/go-travis"
)

func main() {
	client := travis.NewClient(travis.ApiComUrl, "XXXXXXXXXXXXXXXXX")
	builds, _, err := client.Builds.ListByRepoSlug(context.Background(), "x0rzkov/twint-docker", nil)
	if err != nil {
		log.Fatalln(err)
	}
	// pp.Println(resp)
	for _, build := range builds {
		n := structs.Names(build)
		pp.Println("Names", n)
	}
	// Jobs.Cancel is unavailable without authentication
	// _, _, err = client.Jobs.Cancel(context.Background(), 12345)
}
```

```bash
 go run travis.go
2019/12/19 13:33:17 json: cannot unmarshal number -8 into Go struct field Build.builds.duration of type uint
exit status 1
```

Any idea of what could be the problem ?

Cheers,
X.

**Comment by mbyio** [comment]
I also ran into this, unfortunately, I think it is probably an issue in the Travis API.

**Comment by mbyio** [comment]
Actually, looking at the JSON structures, they should not be using `uint` if only because it doesn't specify a size. It should be `uint64` at least.

**Comment by shuheiktgw** [comment]
I'm sincerely sorry for my tardy response. I've started actively maintaining this project again, so give me some time to figure out what is wrong.

**Comment by shuheiktgw** [comment]
How on the earth could duration be negative, haha?
Anyway, as @mbyio suggested, I think I should change the duration to `uint64`. Thanks!

**Comment by shuheiktgw** [comment]
I've released v0.3.0 and fixed this problem, so please use that version or above! 👍 

- https://github.com/shuheiktgw/go-travis/releases/tag/v0.3.0
